### PR TITLE
fix(sumologicexporter): Replace unrecognized source template tokens with `undefined` instead of empty string

### PR DESCRIPTION
**Pull request by astencel-sumo**

### CODE REVIEW

File: pkg/exporter/sumologicexporter/source_format.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func (s *sourceFormat) format(f fields) string {
 		if ok {
 			labels = append(labels, tracetranslator.AttributeValueToString(v))
 		} else {
-			labels = append(labels, "")
+			labels = append(labels, "undefined")
```
Comment (pmalek-sumo) [inline pkg/exporter/sumologicexporter/source_format.go:83]: Make it a `const`?

Comment (astencel-sumo) [reply pkg/exporter/sumologicexporter/source_format.go]: Sure. Does this look OK? https://github.com/SumoLogic/sumologic-otel-collector/pull/88/commits/a7d4d6ae12ab5ff79b842b7f99be220d6497388b

Comment (astencel-sumo) [reply pkg/exporter/sumologicexporter/source_format.go]: Oh I guess I need to use this const in translateConfigValue function too...

Comment (astencel-sumo) [reply pkg/exporter/sumologicexporter/source_format.go]: How about this? https://github.com/SumoLogic/sumologic-otel-collector/pull/88/commits/ad126f294ef470044c2ae627374ce7bc5e9149ff

Comment (pmalek-sumo) [reply pkg/exporter/sumologicexporter/source_format.go]: `unrecognizedAttributeValue`?

Comment (astencel-sumo) [reply pkg/exporter/sumologicexporter/source_format.go]: Done

**Review by pmalek-sumo — COMMENTED** [review]
One thing that this PR is missing is mentioning this behaviour in README

**Comment by astencel-sumo** [comment]
> One thing that this PR is missing is mentioning this behaviour in README

Thanks @pmalek-sumo, good point. Added info in README.

**Review by pmalek-sumo — APPROVED** [review]

**Review by sumo-drosiek — APPROVED** [review]

**Pull request merged**